lametro/events.py

```diff
@@ -12,13 +12,106 @@ class LametroEventScraper(LegistarAPIEventScraper):
     EVENTSPAGE = "https://metro.legistar.com/Calendar.aspx"
     TIMEZONE = "America/Los_Angeles"
 
+    def _pair_events(self, events):
+        paired_events = []
+        unpaired_events = []
+
+        for incoming_event in events:
+            try:
+                partner_event, = [e for e in unpaired_events
+                                  if incoming_event.is_partner(e)]
+            except ValueError:
+                unpaired_events.append(incoming_event)
+
+            else:
+                unpaired_events.remove(partner_event)
+                paired_events.append(incoming_event)
+                paired_events.append(partner_event)
+
+        return paired_events, unpaired_events
+
+    def _find_partner(self, event):
+
+        results = list(self.search('/events/', 'EventId',
+                                   event.partner_search_string))
+        if results:
+            partner, = results
+            partner = LAMetroAPIEvent(partner)
+            assert event.is_partner(partner)
```

Comment:
`partner.is_parter(event)` would work the same and makes more logical sense, I think.

Reply:
For me it's clearer this way. Is this a change that you require?

Reply:
I suppose not, no.